src/test/java/core/basesyntax/FileWorkTest.java

```diff
@@ -21,15 +22,15 @@
             new String[]{"was", "was", "whenever", "which", "which", "worse"};
 
     @Test
-    public void readFromEmptyFile() {
+    public void readFromEmptyFile() throws IOException {
```

Comment:
don't change tests =)

Reply:
done
p.s. was ready for everything to make it work) sorry, I know, it's bad practice